src/app/features/reference-data/reference-data-list/reference-data-list.component.ts

```diff
@@ -53,6 +93,12 @@ export class ReferenceDataListComponent implements OnInit {
     });
   }
 
+  deletedItems(): void {
+    this.router.navigate(['deleted-items'], { relativeTo: this.route }).then(() => {
+      window.location.reload();
```

Comment:
why are we doing a refreshing the page?

Reply:
I've taken this out now, I had just did it the same as the amend and add, but I realise we don't actually need it for the deleted page